### PR TITLE
docs(core): fix typo in interpolation property description

### PR DESCRIPTION
**Pull request by waterplea**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/core/src/metadata/directives.ts [hunk 1]
```diff
@@ -533,7 +533,7 @@ export interface Component extends Directive {
   encapsulation?: ViewEncapsulation;
 
   /**
-   * Overrides the default encapsulation start and end delimiters (`{{` and `}}`)
+   * Overrides the default interpolation start and end delimiters (`{{` and `}}`)
```
Comment (gkalpak) [inline packages/core/src/metadata/directives.ts:536]: :+1:

While you're at it, could you also add the missing period (`.`) at the end? 🙏

**Review by gkalpak — APPROVED** [review]
Good catch!

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Comment by petebacondarwin** [comment]
Merge-assistance: this docs only PR should not need presubmit.

**Pull request closed without merging**